test/Microsoft.Extensions.FileProviders.Composite.Tests/project.json

```diff
@@ -1,9 +1,14 @@
 {
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk",
+    "compile": "../../shared/**/*.cs"
+  },
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Testing": "1.2.0-*",
     "Microsoft.Extensions.FileProviders.Composite": "1.2.0-*",
-    "Microsoft.Extensions.FileProviders.Sources": "1.2.0-*",
+    "Moq": "4.6.38-alpha",
```

Comment:
Is there a feature we are using from this alpha version? I see else where in our repos we're using another version: https://github.com/aspnet/BasicMiddleware/blob/ea0042c685ba2228d0d4b85df41fa45a0b360f00/test/Microsoft.AspNetCore.ResponseCompression.Tests/project.json#L15